src/pages/Login/Login.jsx

```diff
@@ -1,26 +1,74 @@
 import { Link } from 'react-router-dom';
+import { checkAccount, createToken } from 'components/utils/LoginValidation';
+import { useState } from 'react';
 
 function Login() {
+  const initialFormData = {
+    email: '',
+    password: '',
+  };
+
+  const [formData, setFormData] = useState(initialFormData);
+
+  function handleChange(e) {
+    const { name, value } = e.target;
+    setFormData({
+      ...formData,
+      [name]: value.trim(),
+    });
+  }
+
+  function handleLogin(event) {
+    event.preventDefault();
+
+    console.log('formData: ', formData);
+    const loginSuccess = checkAccount(formData);
+
+    if (loginSuccess) {
+      console.log('login success');
+      createToken();
+    } else {
+      console.log('login fail');
+    }
+  }
+
   return (
-    <section>
+    <section className="flex flex-col gap-2 items-center">
       <h1>登入</h1>
-      <form action="submit">
-        <label>
+      <form action="submit" onSubmit={handleLogin} className="flex flex-col">
```

Comment:
the onSubmit will disable `form action="submit"`, so here is unnecessary to use `action` here.